app/src/main/res/drawable/ic_three_dots.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+
```

Comment:
Sorry, I'm a little confused as to how the three dots are added using this file, would you be able to explain it? Thank you!